be/src/runtime/mem_tracker.cpp

```diff
@@ -199,6 +199,15 @@ MemTracker::~MemTracker() {
     if (parent()) {
         unregister_from_parent();
     }
+
+#ifdef DEBUG
+    {
+        std::unique_lock<std::mutex> lock(_child_trackers_lock);
+        DCHECK(_child_trackers.size() == 0)
```

Comment:
```suggestion
        DCHECK(_child_trackers.empty())
```
`empty()` may be more efficient.